api/src/paths/project/{projectId}.ts

```diff
@@ -89,7 +89,7 @@ function getProjectWithDetails(): RequestHandler {
         !getProjectActivitiesSQLStatement ||
         !getProjectClimateInitiativesSQLStatement
       ) {
-        throw new CustomError(400, 'Failed to build SQL statement');
+        throw new CustomError(400, 'Failed to identify user ID');
```

Comment:
```suggestion
        throw new CustomError(400, 'Failed to build SQL statement');
```
Nit: I think the original error message was more appropriate here? This error, at least in its current form, is generic to any issue that might have made any of the 4 sql statements null, so we can only really throw some sort of generic error.